model/src/test/kotlin/utils/FindingsMatcherTest.kt

```diff
@@ -37,8 +37,11 @@ import kotlin.random.Random
 
 private fun Collection<LicenseFindings>.getFindings(license: String) = single { it.license == license }
 
+private const val ROOT_LICENSE_FILE_1 = "a/LICENSE"
+private const val ROOT_LICENSE_FILE_2 = "b/LICENSE"
+
 class FindingsMatcherTest : WordSpec() {
-    private val matcher = FindingsMatcher(FileMatcher("a/LICENSE"))
+    private val matcher = FindingsMatcher(FileMatcher(ROOT_LICENSE_FILE_1, ROOT_LICENSE_FILE_2))
```

Comment:
Note that you might be changing behavior here for all tests that already previously were using `matcher`, and maybe expect it to only match a single file.